django/chat/templates/chat/components/streaming_response.html

```diff
@@ -1,7 +1,6 @@
 <div hx-ext="sse"
      sse-connect="{% url 'chat:chat_response' message_id=message.id %}"
      sse-swap="message"
-     hx-swap="morph:innerHTML"
```

Comment:
Morphing stopped working at some point. I only really had it so that the dots would continue their animation smoothly. Very low priority to fix.